sqlparse/keywords.py

```diff
@@ -842,3 +915,513 @@ def is_keyword(value):
     'IN': tokens.Keyword,
     'LOOP': tokens.Keyword,
 }
+
+# Transact SQL Syntax.
+# See https://technet.microsoft.com/en-us/library/ms189822(v=sql.110).aspx
```

Comment:
SQL 2012? This list may be more up-to date:
[https://docs.microsoft.com/en-us/sql/t-sql/language-elements/reserved-keywords-transact-sql](https://docs.microsoft.com/en-us/sql/t-sql/language-elements/reserved-keywords-transact-sql)